docs/getting-started/README.md

```diff
@@ -1,38 +1,38 @@
 # Mist Getting Started
 
-## What is Hydrosphere Mist
+## What is Hydrosphere Mist?
 
 Hydrosphere Mist is an open-source service for exposing analytics jobs and machine learning models as web services.
 
-With Hydrosphere Mist you can quickly connect Apache Spark programs and machine learning models trained by Apache Spark with end user applications:
+With Hydrosphere Mist, you can quickly connect Apache Spark programs and machine learning models trained by Apache Spark with end user applications:
  - Serve (or score) machine learning models from the REST API with low latency and high throughput
  - Run parametrized Apache Spark jobs from REST API and receive synchronous or asynchronous results
  - Subscribe to streaming jobs and receive reactive stream of results
  - Modify (parametrize) streaming jobs on the fly from the application layer
 
-Our goal is to simplify and facilitate delivery of smart end-to-end solutions on top of machine learning stack.
+Our primary goal is to simplify and facilitate delivery of smart end-to-end solutions on top of machine learning stack.
 
 Hydrosphere Mist is:
- - Resilient: master-slave, auto-recoverable, auto-scalable, self healing
- - Cluster friendly: Kubernetes, DC/OS, ECS, custom
+ - Resilient: master-slave, auto-recoverable, auto-scalable, self=healing
+ - Cluster-friendly: Kubernetes, DC/OS, ECS, custom
  - Hadoop distribution agnostic: Hortonworks, Cloudera, MapR, custom
 
-## Why do I need Hydrosphere Mist
-Hydrosphere helps
+## Why do I need Hydrosphere Mist?
+Hydrosphere extensively assists;
  - Data Scientists and Big Data Engineers to deploy analytics jobs and machine learning models as web services
  - Web developers to tap into analytics using familiar REST or messaging API
  - Managers to facilitate delivery of end-to-end analytics solutions that include data science, data engineering, middleware and application components nicely decoupled and aligned with each other
 
-## Hydrosphere Mist is not:
```

Comment:
@Barbara615 it is better to highlight "is not" in the title, it is pretty common trick, like here https://kubernetes.io/docs/whatisk8s/